rpcs3/Emu/Io/Buzz.cpp

```diff
@@ -27,7 +28,7 @@ void fmt_class_string<buzz_btn>::format(std::string& out, u64 arg)
 	});
 }
 
-usb_device_buzz::usb_device_buzz(u32 first_controller, u32 last_controller, const std::array<u8, 7>& location)
+usb_device_buzz_emu::usb_device_buzz_emu(u32 first_controller, u32 last_controller, const std::array<u8, 7>& location)
```

Comment:
Why are all the devices renamed?
This just adds tons of unnecessary history changes

Reply:
I can undo that, it was something id just copied from that other MR

Reply:
I have reverted that change now.

Reply:
Just pick the necessary changes into a fresh tree. If you're concerned about attribution you can add a co-authored tag to the commit messages to include the original author in git history.

Reply:
Yeah, only the necessary changes are in the commit now